cypress/integration/admin/pageBuilder/blocks/pageBlocks.cy.ts

```diff
@@ -0,0 +1,57 @@
+import { nanoid } from "nanoid";
+import { customAlphabet } from "nanoid";
+
+context("Blocks Page", () => {
```

Comment:
```suggestion
context("Page Builder - Blocks", () => {
```

(per https://www.notion.so/webiny/End-to-end-Testing-With-Cypress-22ca4fdce40b41d89e855651a3cfd249?pvs=4#39054b16ad9241589b8c4ae503a311f7)